typewritten.zsh

```diff
@@ -30,6 +30,7 @@ _set_left_prompt() {
   local return_code="%(?,,%F{red}%? )"
   if [ "$TYPEWRITTEN_DISABLE_RETURN_CODE" = true ]; then
     return_code=""
+    prompt_color = "%(?,%F{blue})"
```

Comment:
`prompt_color="%F{blue}"` would be sufficient here.

The condition syntax used above with the variable `prompt_color` checks for a return code and paints the prompt accordingly.

Side note: be careful the bash that can be very unforgiving with spaces among other things. In this case `prompt_color = ""` would try to call a function named `prompt_color` with the stuff on the right as params, which would trigger errors.

Removing the spaces, like `prompt_color=""` behaves the right way.